examples/plotting/file/sprint.py

```diff
@@ -117,17 +118,17 @@ def selected_name(name, medal, year):
 <div style="font-size: 11px; color: #666;">@{MetersBack}{0.00} meters behind</div>
 """
 
-plot.add_tools(HoverTool(tooltips=tooltips, renderers=[medal]))
+plot.add_tools(HoverTool(tooltips=tooltips, renderers=[medal_circle]))
```

Comment:
we could also pass the tooltips template string to `figure` and then `p.hover.renderers = [...]` that would avoid the `HoverTool` import and the low level call the `add_tools`